logagg/formatters.py

```diff
@@ -150,6 +149,7 @@ def django(line):
                           u'timestamp': 1506061932546,
                           u'user': u'root'},
               'timestamp': '2017-09-22T06:32:15'},
+     'level': 'INFO',
```

Comment:
are we testing in validation function in collector that the values of level ought to be one of 'INFO', 'DEBUG' etc?